Makefile

```diff
@@ -117,3 +137,11 @@ release: clean fmtcheck cross
 	@mkdir -p $(BUILD_DIR)/crc-$(CRC_VERSION)-linux-amd64
 	@cp LICENSE $(BUILD_DIR)/linux-amd64/crc $(BUILD_DIR)/crc-$(CRC_VERSION)-linux-amd64
 	tar cJSf $(RELEASE_DIR)/crc-$(CRC_VERSION)-linux-amd64.tar.xz -C $(BUILD_DIR) crc-$(CRC_VERSION)-linux-amd64
+
+.PHONY: embed_bundle
+embed_bundle: LDFLAGS += $(BUNDLE_EMBEDDED)
+embed_bundle: cross binappend
+	@$(call check_defined, BUNDLE_DIR, "Embedding bundle requires BUNDLE_DIR set to a directory with CRC bundles for all architectures")
+	binappend-cli write $(BUILD_DIR)/linux-amd64/crc $(BUNDLE_DIR)/crc_libvirt_$(BUNDLE_VERSION).$(BUNDLE_EXTENSION)
+	binappend-cli write $(BUILD_DIR)/darwin-amd64/crc $(BUNDLE_DIR)/crc_hyperkit$(BUNDLE_VERSION).$(BUNDLE_EXTENSION)
+	binappend-cli write $(BUILD_DIR)/windows-amd64/crc $(BUNDLE_DIR)/crc_virtualbox$(BUNDLE_VERSION).$(BUNDLE_EXTENSION)
```

Comment:
Same here as above.

Reply:
Windows virtualbox? This is untested and should therefore not be part of this PR as an option

Reply:
We can not make this part of the PR anyway, as this is untested.